### ISSUE TITLE
Build with Visucal Studio Code

### ISSUE THREAD

**Issue by tabeatheunicorn** [janwilmans/copperspice-gettingstarted#2]
It would be nice to know the configurations needed to build the projekt with VS Code or other IDEs

Currently it's not working out of the box

**Comment by janwilmans** [comment]
Thanks for reporting! I did not consider using Visual studio Code (or CLion for that matter), the current examples only work right with Visual studio 2017 or 2019 (but the community edition is free for download if you want to try it)

**Comment by janwilmans** [comment]
I looked into this, it looks like Visual Studio Code does not work with Nuget c++ packages, there is an extension for [.NET Core packages](https://marketplace.visualstudio.com/items?itemName=jmrog.vscode-nuget-package-manager) , but that doesn't help us.

After looking into this for a while I realized that to be able to build with visual studio code, you would still need Visual Studio 2019 installed. (because that is how you get the c++ compiler).
This kindof defeats the purpose of using visual studio code, unless you would prefer it as an environment to the 'real' visual studio.
